src/main/java/com/twilio/ServerApp.java

```diff
@@ -47,9 +48,12 @@ public static void main(String[] args) {
         configuration.put("TWILIO_API_KEY", System.getenv("TWILIO_API_KEY"));
         configuration.put("TWILIO_API_SECRET", System.getenv("TWILIO_API_SECRET"));
         configuration.put("TWILIO_NOTIFICATION_SERVICE_SID", System.getenv("TWILIO_NOTIFICATION_SERVICE_SID"));
-        configuration.put("TWILIO_CHAT_SERVICE_SID",System.getenv("TWILIO_CHAT_SERVICE_SID"));
-        String syncServiceSID = System.getenv("TWILIO_SYNC_SERVICE_SID");
-        configuration.put("TWILIO_SYNC_SERVICE_SID",syncServiceSID != null ? syncServiceSID : "default");
+        Optional<String> chatServiceSid = Optional.ofNullable(System.getenv("TWILIO_CHAT_SERVICE_SID"));
+        if (chatServiceSid.isPresent() && !chatServiceSid.get().isEmpty()) {
+            configuration.put("TWILIO_CHAT_SERVICE_SID", chatServiceSid.get());
+        }
```

Comment:
@ajtack Perhaps you can refactor code into a more readable way.

```.java
Optional.ofNullable(System.getenv("TWILIO_CHAT_SERVICE_SID"))
        .filter(it -> !it.isEmpty())
        .ifPresent(chatServiceSid -> configuration.put("TWILIO_CHAT_SERVICE_SID", chatServiceSid));

String syncServiceSID = Optional.ofNullable(System.getenv("TWILIO_SYNC_SERVICE_SID"))
        .filter(it -> !it.isEmpty())
        .orElse("default"));

configuration.put("TWILIO_SYNC_SERVICE_SID", syncServiceSID);
```

Wdyt? ^^

Reply:
Ah that is indeed nicer. Replaced that commit with f4c246c06059be83486670bc42ab8d86dac4557a.